### ISSUE TITLE
Correct heading order for the Extension page

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1618]
## Problem
On the Extension page (https://search.openverse.engineering/extension), the main heading, 'A browser extension that lets you search and use open-licensed and public domain images.', should be h1.

## Description

The main heading should use `<h1>` element, but the visual styles should match the designs.

**Comment by ramadanomar** [comment]
I'd love to fix this!

**Comment by obulat** [comment]
Hi @ramadanomar 👋, I'm assigning the issue to you. Please note that I've added some details about the implementation in the parent issue, #1614.

**Comment by ramadanomar** [comment]
#1638

**Comment by obulat** [comment]
Closing as fixed by #1644. This issue wasn't closed automatically because that PR was tackling two issues at once.
